Subaharish/HTML BASICS/assignment3.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Registration</title>
+    </head>
+    <body>
+        <h1>Register</h1>
+        <form>
+            First name:
+            <input type="text" name="Firstname"  placeholder="Firstname" required>
+            Last name:
+            <input type="text" name="Lastname" placeholder="Lastname" required>
+            <br>
+            <br>
+            Gender:
+
+            <label for="Male">Male:</label>
+            <input id="Male" type="radio" name="gender" value="male" checked>
```

Comment:
Male shouldn't be checked by default as it can end up in accidental submission and also offensive to other genders in certain cases.